### DISCUSSION TITLE
Website down?

### DISCUSSION THREAD

**Discussion by stvie** [hydrogen-music/hydrogen#1976] [General]
Hi, hydrogen-music.org doesn't load for me, your website might be down. Please delete this if I'm incorrect or this is the wrong place for reporting. Thanks for the great software!

**Comment by oleastre** [comment]
If it can help, I'm able to access it today.

**Comment by theGreatWhiteShark** [comment]
Hey,

Sorry for the late reply. I was on vacation.

> hydrogen-music.org doesn't load for me, your website might be down.

Odd. But thanks for the notice!

We use Github Pages and as such the Github CDN to deliver the website. Maybe some maintenance or DDOSing happened there. Or our registrar had trouble with some of its DNS servers. Don't know. I don't have access to log files of either of them.

It seems to have healed itself. Let's just hope it does not happen again.
